src/index.js

```diff
@@ -5,13 +5,11 @@
  * Here we should add more descriptions
  * Below are all components referenced
  *
- * - See {@link ws-header/ws-header.md|WSHeader}
- * - See {@link ws-date-picker/ws-date-picker.md|WSDatePicker}
- * - See {@link ws-dropdown/ws-dropdown.md|WSDropdown}
- * - See {@link ws-inline-edit/ws-inline-edit.md|WSInlineEdit}
- * - See {@link ws-notification/ws-notification.md|WSNotification}
- * - See {@link ws-week-picker/ws-week-picker.md|WSWeekPicker}
- * - See {@link ws-tiles-chart/ws-tiles-chart.md|WSTilesChart}
```

Comment:
ok, never mind. I think the whole Markdown Index.md was to have a easy to use API documentation hosted on Github. As we move to show them in our styleguide anyway maybe this index.md is unnecessary anymore.